File: app/src/main/java/com/yke/twittershrink/Mood/Document.java
package com.yke.twittershrink.Mood;

import com.google.gson.annotations.SerializedName;

/**
 * Sentiment Information:
 */
public class Document {

    @SerializedName("score")
    private float score;

    public Document(float score) {
        this.score = score;
    }

    public float getScore() {
        return score;
    }
}

File: app/src/main/java/br/com/tweetanalyzer/models/Document.kt
package br.com.tweetanalyzer.models

import com.google.gson.annotations.SerializedName

/**
 * Created by gabrielsamorim
 * on 19/05/18.
 */
data class Document(@SerializedName("content") val text: String,
                    @SerializedName("type") val textType: String = "PLAIN-TEXT")